doc/languages-frameworks/python.section.md

```diff
@@ -2016,6 +2016,10 @@ example of such a situation is when `py.test` is used.
 
 * Tests that attempt to access `$HOME` can be fixed by using the following
   work-around before running tests (e.g. `preCheck`): `export HOME=$(mktemp -d)`
+* Compiling with Cython causes tests to fail with a `ModuleNotLoadedError`.
+  This can be fixed with two changes in the derivation: 1) replacing `pytest` with
+  `pytestCheckHook` and 2) adding a `preCheck` containing `cd $out` to run
```

Comment:
Maybe `cd /` or something should be preferred because we really don't want to import anything that's not in `$PYTHONPATH`.

Reply:
I'm basing this on #255262 and the discussion there as well as fixing two packages' tests with this technique. If there's a better suggestion, I'm all ears.

Reply:
It's indeed important that whatever is installed takes precedence. In that sense, any folder but the source folder is good enough from that point of view. Then, comes the next, which @dotlambda mentions, we don't want to pick up anything more that we don't want either. Therefore, some folder where there is nothing of interest would be preferred.

Note it is also important to switch back directly again (assuming `cd` is used) for later phases/hooks.